tests/test_views.py

```diff
@@ -1504,17 +1515,40 @@ def test_list_services_simple(self, api100, requests_mock, backend1):
         }
 
     def test_list_services_no_supporting_backends(
-        self, api100, requests_mock, backend1
+        self, api100, requests_mock, backend1, caplog
     ):
         """None of the upstream backends supports secondary services"""
+        caplog.set_level(logging.ERROR)
         api100.set_auth_bearer_token(TEST_USER_BEARER_TOKEN)
+
+        # No backends supported ==> no mock for capabilities at "GET /"
```

Comment:
>  No backends supported ==> no mock for capabilities at "GET /"

This is bit misleading comment:
`GET /` should always be supported (it's the bare minimum an openEO backend should provide)
Note that the `backend1` fixture provides such a minimal mock.

Reply:
Corrected the comment.